presto-orc/src/main/java/com/facebook/presto/orc/reader/ByteSelectiveStreamReader.java

```diff
@@ -80,6 +81,9 @@
     private boolean allNulls;
     private boolean valuesInUse;
 
+    private byte[] nonNullValueTemp = new byte[0];
```

Comment:
Let's find another word than "temp". Depends on the use case, it could be buffer, pool, reusable, etc.

Reply:
Sure. How about `nonNullValueBuffer` ?